attributes/default.rb

```diff
@@ -67,7 +51,7 @@
 # This specifies how to connect to a master or a quorum of masters. This flag
 # works with 3 different techniques.  See Mesos documentation for details:
 # http://mesos.apache.org/documentation/latest/configuration
-default['mesos']['slave']['flags']['master']                = '`cat /etc/mesos/zk`'
+default['mesos']['slave']['flags']['master']                = 'localhost:5050'
```

Comment:
Well, since ZooKeeper is not necessary (albeit highly recommended) for Mesos, I have opted for the default option to connect to itself as to make the cookbook more independent. However, I wholeheartedly agree with you that people should use ZooKeeper for discovering masters.